I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/capitulo2.rpy", line 358, in script
    jump cap3
ScriptError: could not find label 'cap3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/capitulo2.rpy", line 358, in script
    jump cap3
  File "C:\Users\samue\Downloads\renpy-8.3.7-sdk\renpy\ast.py", line 1712, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\samue\Downloads\renpy-8.3.7-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'cap3'.

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.7.25031702
Proj1 Beta 0.1
Fri Jun 27 20:46:34 2025
